applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -104,11 +146,27 @@ static void stream_qos_set_cb(struct bt_audio_stream *stream)
 static void stream_enabled_cb(struct bt_audio_stream *stream)
```

Comment:
This is a callback from BLE RX I believe, should not do a k_sleep in  a while in it. Maybe move the retrying of the connection to a k_work?